presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationSmokeTest.java

```diff
@@ -116,6 +121,148 @@ private KafkaTopicFieldDescription createOneFieldDescription(String name, Type t
         return new KafkaTopicFieldDescription(name, type, mapping, null, dataFormat, null, false);
     }
 
+    @Test(dataProvider = "testRoundTripAllFormatsDataProvider")
+    public void testRoundTripAllFormats(RoundTripTestSetup roundTripTestSetup)
+    {
```

Comment:
Don't you need to create a topic here?

Reply:
I didn't include the actual test cases because the encoder formats aren't included in this pr. Maybe it would be best to not include this test in this pr either.